index.js

```diff
@@ -163,7 +163,7 @@ class NodeStl {
     // regardless of this standard, so we check if offset 80, the location of
     // the number of triangles in a binary STL matches the expected file size.
 
-    expected_size = header_size + n_faces * face_size;
```

Comment:
In the current master the `header_size` needs to be renamed in L143 https://github.com/johannesboyne/node-stl/blob/master/index.js#L143 as well, weird that github is not showing it correctly here.

Reply:
indeed